.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "FV3"]
 	path = FV3
-	url = https://github.com/NOAA-EMC/fv3atm
-	branch = develop
+	url = https://github.com/YihuaWu-NOAA/fv3atm
+	branch = master
```

Comment:
FV3 submodule should point to your `develop` branch, not master. All your changes in your fv3atm fork are in develop branch.